lib/rules/jsx-indent.js

```diff
@@ -218,41 +220,47 @@ module.exports = {
       }
     }
 
+    function getOpeningElementIndent(node) {
```

Comment:
Needed to move this because the `JSXClosingElement` needs to get the opening element's indent as it _should be_ not as it is.